src/m365/entra/commands/m365group/m365group-set.ts

```diff
@@ -86,12 +99,32 @@ class EntraM365GroupSetCommand extends GraphCommand {
       },
       {
         option: '-l, --logoPath [logoPath]'
+      },
+      {
+        option: '--allowExternalSenders [allowExternalSenders]',
+        autocomplete: ['true', 'false']
+      },
+      {
+        option: '--autoSubscribeNewMembers [autoSubscribeNewMembers]',
+        autocomplete: ['true', 'false']
+      },
+      {
+        option: '--hideFromAddressLists [hideFromAddressLists]',
+        autocomplete: ['true', 'false']
+      },
+      {
+        option: '--hideFromOutlookClients [hideFromOutlookClients]',
+        autocomplete: ['true', 'false']
+      },
+      {
+        option: '--isSubscribedByMail [isSubscribedByMail]',
```

Comment:
Seems like we have to use another endpoint to update this property. At least, it doesn't work for me to update it with this command. Is it for you? If not, let's just drop this option then.

Reply:
Correct